modules/oe_translation_poetry/src/Logger/PoetryLoggerChannelFactory.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+namespace Drupal\oe_translation_poetry\Logger;
+
+use Drupal\Core\Logger\LoggerChannelFactory;
+
+/**
+ * Custom logger channel factory for the Poetry client.
+ *
+ * Since we need a custom logger channel class to process Poetry messages
+ * we also need a custom logger channel factory that will instantiate said
+ * custom logger when prompted.
+ *
+ * @see \Drupal\oe_translation_poetry\Poetry
+ * @see \Drupal\oe_translation_poetry\Logger\PoetryLoggerChannel
+ */
+class PoetryLoggerChannelFactory extends LoggerChannelFactory {
+
+  /**
+   * {@inheritdoc}
+   */
+  public function get($channel) {
+    if ($channel !== 'poetry') {
```

Comment:
This conditional is not needed. We can use multiple channel names for this, no issue. We are the only ones who will use this channel factory.